.gitignore

```diff
@@ -82,6 +82,9 @@ test-reports/
 /test_bash.sh
 /python_test_out.txt
 
+# Auto generated doc
+_sidebar.rst.inc
+
```

Comment:
thank you! I have been meaning to add this bloody sidebar to ignore for a while now :grin: